jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/JpaMedia.java

```diff
@@ -21,7 +21,7 @@
  * Created by Jedd Cuison on 12/6/2016
  */
 @Entity(name = "media")
-@NamedQueries({@NamedQuery(name = JpaMedia.QUERY_FIND_BY_NAME, query = "SELECT m FROM media m WHERE m.name = :name")})
+@NamedQueries({@NamedQuery(name = JpaMedia.QUERY_FIND_BY_NAME, query = "SELECT m FROM media m WHERE lower(m.name) = lower(:name)")})
```

Comment:
Should this query not also include the media type? Looking at this query you can not have a JDK with name 'Peter' and a Tomcat with 'Peter' ??